pkg/reconcilermanager/controllers/reconciler_base_test.go

```diff
@@ -601,8 +604,21 @@ func manualChangeResourceLimits(unDep *unstructured.Unstructured, isManaulChange
 	if !isManaulChange {
```

Comment:
I'd suggest moving this `if` statement up to the unit test to make it clear what this does when reading the test. It's generally a best practice to minimize logic in helper functions for unit tests. Ditto for the `isOverride` if statement in the above function

Reply:
Sure, moved the logic out of the function.